examples/react-native/package.json

```diff
@@ -7,7 +7,7 @@
     "test": "jest"
   },
   "dependencies": {
-    "react": "16.2.0",
+    "react": "16.0.0",
```

Comment:
peer dep warning was annoying

Reply:
Can you downgrade to 16.0 but use a caret instead?

Reply:
Then we'll fail the peer dep again, won't we? Or does yarn check peer deps when resolving the tree?

Reply:
If I add a caret, we're back to the peer dep warning.

React-native has no range on its peer: https://github.com/facebook/react-native/blob/33a2e533b76d35c1b9fb32e926f7c2c27cb616e9/package.json#L144 (it's been updated to 6.2.0, but that's not released)